### ISSUE TITLE
Support for --parameter-values in 0.3.0

### ISSUE THREAD

**Issue by metaskills** [awslabs/aws-sam-cli#456]
I read issue #400 and it seems that SAM local no longer supports the `--parameter-values` CloudFormation parameter now. Just want to make sure this is intentional and that the use of `--env-vars` is the preferred replacement when running locally? 

I have not tested if actions like `deploy` support `--parameter-values` still. Just wondering what the general direction is with regard to these configs. Thanks in advance!

**Comment by sanathkr** [comment]
yes, `--env-vars` is the recommended solution for `sam local` suite of commands. But for `deploy`, `--parameter-values` is still available and won't be deprecated.

Does this unblock you?

**Comment by metaskills** [comment]
> Does this unblock you?

Thank you it does. That distinct will help me upgrade to v0.3.0 once this https://github.com/awslabs/aws-sam-cli/issues/457 is resolved. Thanks again! Closing this issue now too.

**Comment by onelaview** [comment]
Sadly that Lambda in AWS China (cn-north-1, for example) still doesn't support environment variables yet so `--env-vars` is not an applicable solution here. Please re-consider adding support for `--parameter-values` or `--parameter-overrides` argument to be able to pass a parameter to CloudFormation file for `sam local invoke` command.

**Comment by jfuss** [comment]
@onelaview Soon! We have support for `--parameter-overrides` merged into develop (#657). This adds support for SAM CLI to resolve Refs of Parameters and pick up default values of those Parameters. It will be available in the next release. We will be closing #572 once this is available. Stay tuned! :)
